src/main/java/it/pagopa/ecommerce/commons/domain/TransactionActivationRequested.java

```diff
@@ -23,8 +25,11 @@
  * @see BaseTransaction
  */
 @EqualsAndHashCode(callSuper = true)
+@Getter
 public final class TransactionActivationRequested extends BaseTransaction implements Transaction {
 
+    private OriginType originType;
```

Comment:
This should be on `BaseTransaction` so that all other transaction domain classes can have access to `originType` :)

Reply:
you're right, fixed ;)